templates/_macros/vf_cta-section.jinja

```diff
@@ -0,0 +1,105 @@
+# All Params
+  # title_text: H2 title text
+  # link_text: H2 Link text
+  # link_href: href for the link text
+  # layout: Layout type of cta section. Options are "cta-default-100", "cta-default-25-75", "cta-block-100", "cta-block-25-75". Default is "cta-default-100".
```

Comment:
I wonder if we need to namespace these layout parameters with `cta`, it seems to increase the number of characters you need to type without much other benefit..